source/fundamentals/connection/connect.txt

```diff
@@ -63,9 +63,15 @@ instance on port ``27017`` of ``localhost``:
 .. literalinclude:: /includes/fundamentals/code-examples/connection/LocalConnection.cs
    :language: csharp
 
-.. tip::
+.. tip:: Reuse Your Client
+
+   As each ``MongoClient`` represents a pool of connections to the
+   database, most applications only require a single instance of a
```

Comment:
```suggestion
   Because each ``MongoClient`` represents a pool of connections to the
   database, most applications require only a single instance of a
```